commit 29e7104ddeef72c01f897b68ddd2690e2a108cfb
Author: Radoslaw Karabowicz <r.karabowicz@gsi.de>
Date:   Mon Jan 18 14:12:39 2021 +0100

    Fix the error in ertrak.F
    
    Fix the problem with string as function argument.
    The error was causing wrong string recognition, and, occasionally,
    a segmentation violation.

diff --git a/erdecks/ertrak.F b/erdecks/ertrak.F
index 3fb0813..fa95cb1 100644
--- a/erdecks/ertrak.F
+++ b/erdecks/ertrak.F
@@ -78,8 +78,8 @@
 *     
 * *** Decode character option
 *
-      CHOPTI = CHOPT
-      CALL UOPTC (CHOPT, 'BELMOPVX', IOPT)
+      CHOPTI = CHOPT(1:len(CHOPT))
+      CALL UOPTC (CHOPTI, 'BELMOPVX', IOPT)
 *
       IF (IOPTB.EQ.0) THEN
          BACKTR = 1.
